chalicelib/checks/audit_checks.py

```diff
@@ -425,6 +425,9 @@ def check_help_page_urls(connection, **kwargs):
         for url in urls:
             if url.startswith('mailto'):
                 continue
+            elif 'biorxiv.org'.lower() in url:
```

Comment:
should this be `elif 'biorxiv.org' in url.lower():`? biorxiv.org is already lowercase.

Reply:
That's what I meant, thanks for catching that!